uniffi_bindgen/src/interface/record.rs

```diff
@@ -131,9 +131,6 @@ impl APIConverter<Field> for weedle::dictionary::DictionaryMember<'_> {
             bail!("dictionary member attributes are not supported yet");
         }
         let type_ = ci.resolve_type_expression(&self.type_)?;
-        if let Type::Object(_) = type_ {
-            bail!("Objects cannot currently appear in record fields");
-        }
```

Comment:
Does this mean you just need to write tests in Kotlin Swift and Ruby to prove to yourself that it works as documented?

Reply:
> Does this mean you just need to write tests in Kotlin Swift and Ruby to prove to yourself that it works as documented?

I suspect the bindings themselves might also need work - eg, CI is showing that the kotlin code being generated here has a syntax error.

Reply:
I haven't done ruby - I think python, swift and kotlin not needing any changes demonstrates it works ok.